src/features/organizations/store.ts

```diff
@@ -46,6 +46,40 @@ const OrganizationsSlice = createSlice({
       state.eventsByOrgId[orgId] = remoteList(events);
       state.eventsByOrgId[orgId].loaded = new Date().toISOString();
     },
+    orgFollowed: (state, action: PayloadAction<ZetkinMembership>) => {
+      const membership = action.payload;
+
+      const existingMembership = state.userMembershipList.items.find(
+        (item) => item?.data?.organization.id === membership.organization.id
+      );
+
+      if (existingMembership?.data) {
+        existingMembership.data.follow = true;
+        existingMembership.loaded = new Date().toISOString();
+      } else {
+        const membershipWithId: ZetkinMembership & { id: number } = {
+          ...membership,
+          follow: true,
+          id: membership.organization.id,
+        };
+
+        state.userMembershipList.items.push(
+          remoteItem(membership.organization.id, { data: membershipWithId })
```

Comment:
This item is missing `loaded`.

```suggestion
          remoteItem(membership.organization.id, { data: membershipWithId, loaded: new Date().toISOString() })
```

Reply:
Oops! Thanks for spotting it!